src/components/Router.tsx

```diff
@@ -1,17 +1,21 @@
 import React, { FC } from 'react'
-import { Route, Switch } from 'react-router-dom'
+import { Route, Switch, Redirect } from 'react-router-dom'
 import { Teleop } from 'components/pages/Teleop'
 import { Arm } from 'components/pages/Arm'
 import { MapPage } from 'components/pages/MapPage'
 import { ConfigPage } from 'components/pages/Config/ConfigPage'
 
 export const Router: FC = () => {
   return (
-    <Switch>
-      <Route path="/teleop" component={Teleop} />
-      <Route path="/arm" component={Arm} />
-      <Route path="/map" component={MapPage} />
-      <Route path="/config" component={ConfigPage} />
-    </Switch>
+    <>
+      <Redirect exact from="/" to="/teleop" />
+
+      <Switch>
+        <Route path="/teleop" component={Teleop} />
+        <Route path="/arm" component={Arm} />
+        <Route path="/map" component={MapPage} />
```

Comment:
Map page? Did we define the different tabs for the UI? If not it might be best to take the time and define them.

Reply:
The tabs are defined in src/components/Header.tsx this was just a temporary router. I forgot to remove that route